client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,70 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { TextControl, Button, ToggleControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+import apiFetch from '@wordpress/api-fetch';
+
+class SubscriptionEdit extends Component {
+	render() {
+		const { attributes, className, isSelected, setAttributes } = this.props;
+		const { subscribe_placeholder, show_subscribers_total, subscriber_count_string } = attributes;
```

Comment:
These are lowercase to keep them identical with existing shortcode attributes, right?
I think it would still be valuable to look at keeping them as CamelCased just to have codebase more consistent across all blocks and the Calypso codebase in general.

Not sure how to go about transforming them in `settings.attributes` though. 🤔